cmd/vsphere-xcopy-volume-populator/internal/vmware/client.go

```diff
@@ -72,7 +72,7 @@ func (c *VSphereClient) RunEsxCommand(ctx context.Context, host *object.HostSyst
 	return res.Values, nil
 }
 
-func (c *VSphereClient) GetEsxByVm(ctx context.Context, vmName string) (*object.HostSystem, error) {
+func (c *VSphereClient) GetEsxByVm(ctx context.Context, vmId string) (*object.HostSystem, error) {
 	finder := find.NewFinder(c.Client.Client, true)
 
 	//FIXME - need to trace the VM by the datastore, which we should have because
```

Comment:
fix that already - maybe we donh't need to set a datacenter if we get the vm by id. anyhow, get rid of the datacednter part because there may be a case where there is no datacenger